crates/top/rerun/src/commands/entrypoint.rs

```diff
@@ -894,6 +912,7 @@ fn run_impl(
                         startup_options,
                         cc,
                         re_viewer::AsyncRuntimeHandle::new_native(tokio_runtime_handle),
+                        (command_sender, command_receiver),
```

Comment:
I think we should solve this differently. Instead of passing in channels, why not tell send commands to the app from the outside
`re_viewer::App` already exposes a bunch of things, why not add an ability to send commands to a live app. This is also super useful for anyone hosting rerun in their own rust application and wanting to instrument it

Reply:
What do you mean by "add ability to send commands to a live app"? What would that look like?

I agree that this isn't a great solution. I tried to minimizee the changes to the flow of `entrypoint` here. I would personally much prefer constructing the `App` first, then parsing the data sources, and finally adding them all as receivers, instead of the opposite. Then it would always be possible to give the data sources access to the command sender.

Reply:
We could have a separate channel for `DataSourceCommand`s, and the receiver gets passed to the Viewer, which would read from it like it does from smart channels. Not sure if that's what you meant.

Reply:
(we decided to refactor this later)
* https://github.com/rerun-io/rerun/issues/9113